src/ISADotNet.XLSX/CollectionAux.fs

```diff
@@ -104,7 +104,11 @@ module internal List =
 
 module internal Dictionary =
 
-    let tryGetValue k (dict:System.Collections.Generic.Dictionary<'K,'V>) = 
+    let tryGetValue k (dict:System.Collections.Generic.Dictionary<'K,string>) = 
         let b,v = dict.TryGetValue(k)
-        if b then Some v
-        else None
+        // Only get value if 
+        if b && v.Trim() <> ""
```

Comment:
If I see it correctly this will only trim the string it does not only contain whitespace, right? I think a string containing only whitespace should be trimmed too?

For this the second condition `(v.Trim() <> "")` should be removed

Reply:
Nvm the `None` does the job